src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -98,7 +98,19 @@ private void LoadingStatusIndicator_PropertyChanged(object sender, PropertyChang
             });
         }
 
-        public bool CheckBoxesEnabled { get; set; }
+        private bool _checkBoxesEnabled;
```

Comment:
Move instance variable declaration to top

Reply:
fixed